src/pages/Detail.tsx

```diff
@@ -0,0 +1,207 @@
+type PosterInfo = {
+  title: string;
+  updatedAt: string;
+  fullName: string;
+  description: string;
+  _id: string;
+  isOnline: boolean;
+};
+
+type CommentInfo = {
+  _id: string;
+  comment: string;
+  fullName: string;
+  isOnline: boolean;
+};
+
+type SearchPosterResponse = {
+  posterInfo: PosterInfo;
+  commentInfo: CommentInfo[];
+  likeCount: number;
+};
+
+import { useEffect, useState } from 'react';
+import {
+  Box,
+  Flex,
+  Text,
+  Image,
+  Input,
+  Stack,
+  Center,
+  Divider,
+  WrapItem,
+  Drawer,
+  DrawerOverlay,
+  DrawerContent,
+  DrawerCloseButton,
+  DrawerHeader,
+  DrawerBody,
+  Container,
+  AspectRatio,
+  useBoolean,
+} from '@chakra-ui/react';
+import Comment from '../components/common/Comment';
+import TemperatureBar from '../components/common/TemperatureBar';
+import PrimaryAvatar from '../components/common/PrimaryAvatar';
+import PrimaryButton from '../components/common/PrimaryButton';
+import RecommendButton from '../components/common/RecommendButton';
+import { formatDate } from '../utils/Date';
+import { useParams } from 'react-router-dom';
+import { searchPoster } from '../apis/search';
+const Detail = () => {
+  const [data, setData] = useState<SearchPosterResponse | null>(null);
+  const [comments, setComments] = useState<CommentInfo[] | null>(null);
+  const { postId } = useParams<{ postId: string }>();
+  const [isDrawerOpen, setIsDrawerOpen] = useBoolean();
+  useEffect(() => {
```

Comment:
```suggestion
  const [isDrawerOpen, setIsDrawerOpen] = useBoolean();
  
  useEffect(() => {
```

줄 바꿈 해주시면 보기 편할 것 같아요!!